contacts.routing.yml

```diff
@@ -28,3 +28,47 @@ contacts.add_org_form:
     _permission: 'add contacts'
   options:
     _admin_route: TRUE
+
+contacts.dash_block_add:
+  path: '/contacts_add_block/{tab}/{block}/{region}/{weight}'
+  defaults:
+    _controller: '\Drupal\contacts\Controller\DashboardController::addBlock'
+    region: 'left'
+    weight: 0
+  options:
+    no_cache: TRUE
+  requirements:
+    _access: 'TRUE'
+#    _csrf_token: 'TRUE'
+
+contacts.dash_block_move:
+  path: '/contacts_move_block/{tab}/{block}/{region}/{weight}'
+  defaults:
+    _controller: '\Drupal\contacts\Controller\DashboardController::moveBlock'
+    region: 'left'
+    weight: 0
+  options:
+    no_cache: TRUE
+  requirements:
+    _access: 'TRUE'
+#    _csrf_token: 'TRUE'
+
+contacts.dash_block_remove:
+  path: '/contacts_remove_block/{tab}/{block}'
+  defaults:
+    _controller: '\Drupal\contacts\Controller\DashboardController::removeBlock'
+  options:
+    no_cache: TRUE
+  requirements:
+    _access: 'TRUE'
+#    _csrf_token: 'TRUE'
+
+contacts.dash_block_title:
+  path: '/contacts_update_block/{tab}/{block}'
+  defaults:
+    _controller: '\Drupal\contacts\Controller\DashboardController::updateBlockTitle'
+  options:
+    no_cache: TRUE
+  requirements:
+    _access: 'TRUE'
+#    _csrf_token: 'TRUE'
```

Comment:
Having 4 end points for updating a single config I think is a real smell. I think, even stronger than before, that this should be consolidated and we should have a single endpoint for a tab which merges in all the relevant details.